docs/oidc.md

```diff
@@ -20,6 +22,91 @@ Sigstore runs a federated OIDC identity provider, Dex. Users authenticate to the
 
 See [this example](https://github.com/sigstore/fulcio/pull/890), although it is out of date as you'll now need to create an issuer type.
 
+### How to pick a SAN
+
+SANs are important for users to describe identities across platforms. They are
+used in verification policies as the primary identifier for a workload.
+
+Unfortunately there's no one size fits all answer for how to pick the best SAN
+to use for your service. To help, here are a few things to consider when making
+this choice:
+
+- How will users want to query / write policy for artifacts?
+
+  Consider what resource(s) users will want to query against. How would they
+  distinguish resources between different teams? Production vs staging?
+
+  💡 Litmus test: what value is appropriate for
+  `cosign verify --certificate-identity=<?>`
+
+- What's the most-specific identifier that can describe the workload?
+
+  Choosing a SAN is often similar to figuring out what service account your
+  workloads should have. Too broad, you may give unintended access to workloads
+  that don't need it. Too narrow, and you end up having to manage the same
+  permissions across multiple accounts.
+
+- Will the identifier change per-instance?
+
+  Identifiers that are based on UIDs and can change each instance do not make
+  good SANs. They tend to be too narrow and make it difficult to write a policy
+  that will work consistently. If you need to reach for a regex for most
+  policies, your SAN is probably too specific.
+
+- Can the identifier collide with other resources?
+
+  SANs should be unique for the issuer. Resources should not have the ability to
+  use or craft a SAN of another resource.
+
+- Is the identifier well-defined?
+
+  All SANs for a provider should be defined and documented. If an issuer has the
+  ability to produce different SANs, differences and conditions for these SANs
+  should be documented.
+
+#### Case study: GitHub Actions
+
+GitHub Actions uses the `job_workflow_ref` as it's SAN. This has a few nice
+properties when working with GitHub Actions:
+
+- It's tied a class of Job that have a set of permissions.
```

Comment:
```suggestion
- It's tied to a class of Job that have a set of permissions.
```

Reply:
What is meant by this? That there’s a well defined set of access control associated with this job so you know something about its provenance?